### PR TITLE
[Needs Review] b07902045 呂紹齊

### PR DESCRIPTION
**Pull request by archielu**
A Pull Request (PR) should explain why we need this PR and how you implement it, so others could understand it quickly.
Since this is just a homework, you do not need to explain why and how. 

**Do NOT change the status to NeedsReview if you do not want to be graded yet.**

Please fill in the following questions as feedback. It is totally optional and would not be graded.

## Is there any suggestion?

<br />

### CODE REVIEW

File: src/students/b07902045.py [hunk 1]
```diff
@@ -0,0 +1,414 @@
...
+
+    # TODO: download the image from img_url with the request module
+    # and add your student ID on it with draw_text() in the utils module
+    # under src/.
+    from PIL import Image
+    from io import BytesIO
+    import utils
+    r = requests.get(img_url)
+    result_img = Image.open(BytesIO(r.content))
+
```
Comment (amjltc295) [inline src/students/b07902045.py]: You did not write your student ID here

Comment (archielu) [reply src/students/b07902045.py]: ok i change another file

Comment (archielu) [reply src/students/b07902045.py]: @amjltc295

File: src/students/b07902045.py [hunk 2]
```diff
@@ -395,10 +395,15 @@ def task_8(
     # under src/.
     from PIL import Image
     from io import BytesIO
-    import utils
+    from utils import draw_text
     r = requests.get(img_url)
     result_img = Image.open(BytesIO(r.content))
-b07902045
+    text = ImageDraw.Draw(result_img)
+    text.ink = 0 + 0*256 + 256*256*256
```
Comment (amjltc295) [inline src/students/b07902045.py]: Where did you learn this syntax? Did you write this by yourself?

Comment (archielu) [reply src/students/b07902045.py]: @amjltc295 
I learned it from google and I refer to others code
Now, I use draw_text from utils.py in my new file.

Comment (amjltc295) [reply src/students/b07902045.py]: @archielu  Do you have the link? Who did you reference?

Comment (archielu) [reply src/students/b07902045.py]: NO, I find it in
 other classmates' code

Comment (archielu) [reply src/students/b07902045.py]: @amjltc295

**Pull request closed without merging**